### PR TITLE
[WIP] Issue 76

### PR DESCRIPTION
**Pull request by garricn**
This PR will deliver #280.

- [ ] Update status of SE-0180
- [ ] Remove Empty Sections
- [ ] Add Starter Tasks

### CODE REVIEW

**Comment by jessesquires** [comment]
hey @garricn -- just fyi, I updated this branch and added a sponsor. 😊 you'll need to `git pull` if you are working locally

**Comment by garricn** [comment]
I finally had a chance to start writing the draft. Let me know your thoughts. I'm kind of nervous after @gregheo 's amazing haiku 😜

**Comment by jessesquires** [comment]
@garricn -- it looks awesome! 🙌 i push one commit with minor tweaks, mostly just adding a few more links 😊

**Comment by garricn** [comment]
Any idea why Travis/pr failed? Not sure I understand the log.

**Comment by BasThomas** [comment]
Not sure, Danger failed to run. I've restarted the build, hope it was a hiccup?

File: _drafts/2017-06-29-issue-76.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Swift Unwrapped
+
+In [Episode 17: Testing in Swift](https://spec.fm/podcasts/swift-unwrapped/70319), Jesse and JP discuss how Swift and its set of open source tools are tested to ensure that every release breaks as little as possible 😜
+
+### News and community
+
+Matt Godbolt's Compiler Explorer now [supports Swift](https://twitter.com/Catfish_Man/status/877991651548975104).
+
+David Owens [released a preview](https://owensd.io/2017/06/02/apous-early-preview/) for an extension to Visual Studio Code that supports Swift.
```
Comment (garricn) [inline _drafts/2017-06-29-issue-76.md]: A lot of people covered this and this section is getting big. Thoughts on removing it? Or is it all good... @jessesquires

**Comment by BasThomas** [comment]
On Travis failing: it seems like a proselint issue: see https://github.com/SwiftWeekly/swiftweekly.github.io/pull/281#issuecomment-310440414. I'll open an issue over at https://github.com/dbgrandi/danger-prose.

**Comment by garricn** [comment]
I should have a near final draft done by end of day. I would love to get feedback until then before publishing tomorrow :)

File: _drafts/2017-06-29-issue-76.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    heading: Ramp up your skills &mdash; and your career
+    body: Go from junior to senior developer in just a week, with the experts who have taught iOS from the very beginning.
+    displaylink: bignerdranch.com
+---
+
+It was a busy week on the main apple/swift repo. Here are some stats from GitHub Insights:
+
+> Excluding merges, 42 authors have pushed 187 commits to master and 345 commits to all branches. On master, 397 files have changed and there have been 12,419 additions and 9,248 deletions.
+
+It's great to see so much work being done so soon after WWDC. The Core Team and Core Contributors are making significant progress on the road to the offical Swift 4 launch.
```
Comment (modocache) [inline _drafts/2017-06-29-issue-76.md]: Is "Core Contributors" accurate terminology? As far as I know, there's a Swift core team (with some disagreement on whether to capitalize "Core Team"), and there are contributors with commit access, but I'm not aware of a group of "Core Contributors".

File: _drafts/2017-06-29-issue-76.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Philippe Hausler [merged a pull request](https://github.com/apple/swift/pull/10584) that fixes [SR-5292](https://bugs.swift.org/browse/SR-5292). This fixes a crasher in Foundation when working with slices of slices.
+
+Joe Groff [merged a pull request](https://github.com/apple/swift/pull/10556) that adds support for optional chaining / force unwrapping when dealing with KeyPaths.
+
+Doug Gregor [merged a pull request](https://github.com/apple/swift/pull/10565) that improves the handling of concrete types, type aliases, and recursion, and which fixes [SR-4295](https://bugs.swift.org/browse/SR-4295), [SR-4757](https://bugs.swift.org/browse/SR-4757), [SR-4786](https://bugs.swift.org/browse/SR-4786), [SR-5014](https://bugs.swift.org/browse/SR-5014), and [SR-4737](https://bugs.swift.org/browse/SR-4737).
+
+Dave Abrahams opened a [pull request](https://github.com/apple/swift/pull/9806) that implements [SE-0180](https://github.com/apple/swift-evolution/blob/master/proposals/0180-string-index-overhaul.md), the String index Overhaul (See below).
+
+The Swift Man Page [got an update](https://github.com/apple/swift/pull/10241).
```
Comment (modocache) [inline _drafts/2017-06-29-issue-76.md]: Sorry, sorry, another capitalization nit-pick: I think this should be "man page", not "Man Page". Even clearer, I think, would be "The manual displayed when running `man swift` on the command line has been updated."

File: _drafts/2017-06-29-issue-76.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Maxim Moiseev [merged a pull request](https://github.com/apple/swift/pull/9466) that fixes backward compatibility for flatMap on [String]. Check out this related [Swift Puzzle by Robert Widmann](https://twitter.com/codafi_/status/878330155642396673).
+
+David Farler [merged a pull request](https://github.com/apple/swift-clang/pull/95) that upstreams previous Apple Internal index-while-building changes released in the Xcode 9 timeframe. From Brian Gesiak on these changes:
+
+> Apple developed index-while-building on their internal Clang fork in time for Xcode 9. The changes are being merged to swift-clang, the Swift project's fork of Clang, in apple/swift-clang. 87 files changed, 9,127 lines added, 30 lines deleted!. As far a I can tell, there isn't a diff up for this on the upstream Clang repository..."
```
Comment (modocache) [inline _drafts/2017-06-29-issue-76.md]: I don't think the quote from me adds anything here. I think this paragraph reads better as just "David Farler merged a pull request that upstreams previous Apple Internal index-while-building changes released in the Xcode 9 timeframe.", and that's it. :)

**Comment by garricn** [comment]
@modocache Thank you for those excellent comments!

**Comment by garricn** [comment]
@BasThomas Also, what should I do if Prose is still failing?

**Comment by BasThomas** [comment]
Can you merge right now, or not? If not, ping me / any of the owners (https://github.com/orgs/SwiftWeekly/people) and we can merge the PR for you. 👍

Be aware that for publishing, the draft should be moved to the `_posts` folder.

**Comment by BasThomas** [comment]
On starter tasks: we have had some issues without any starter tasks, so you can remove it when there are none.

**Comment by jessesquires** [comment]
@garricn this looks so great! 🙌 😎

**Comment by jessesquires** [comment]
Looks like the draft is good to go, yeah? I removed the Starter Tasks and unused proposals sections.

I'm going to go ahead and merge this. 😄

**Pull request merged**